openslides/core/projector.py

```diff
@@ -63,50 +62,43 @@ def validate_config(cls, config_data):
         """
         if not isinstance(config_data.get('countdown_time'), (int, float)):
             raise ProjectorException('Invalid countdown time. Use integer or float.')
-        if config_data.get('status') not in ('running', 'stop'):
-            raise ProjectorException("Invalid status. Use 'running' or 'stop'.")
+        if not isinstance(config_data.get('running'), bool):
+            raise ProjectorException("Invalid running status. Has to be a boolean.")
         if config_data.get('default') is not None and not isinstance(config_data.get('default'), int):
             raise ProjectorException('Invalid default value. Use integer.')
 
     @classmethod
-    def control(cls, action, projector_id=1, index=0):
-        """
-        Starts, stops or resets the countdown with the given index on the
-        given projector.
-
-        Action must be 'start', 'stop' or 'reset'.
-        """
+    def control(cls, action):
         if action not in ('start', 'stop', 'reset'):
-            raise ValueError("Action must be 'start', 'stop' or 'reset', not {}.".format(action))
-
-        projector_instance = Projector.objects.get(pk=projector_id)
-        projector_config = {}
-        found = False
-        for key, value in projector_instance.config.items():
-            if value['name'] == cls.name:
-                if index == 0:
+                raise ValueError("Action must be 'start', 'stop' or 'reset', not {}.".format(action))
+
+        # Use the first countdown if there is one
```

Comment:
It is used in agenda/models.py

It will always take the first countdown, not all. This was a wish from Emanuel. Currently if no countdown is found it does nothing. I will implement, that a countdown will be created.